std/math.d

```diff
@@ -2751,8 +2751,7 @@ if (isFloatingPoint!T)
     int exp;
     real mantissa = frexp(123.456L, exp);
 
-    // check if values are equal to 19 decimal digits of precision
-    assert(equalsDigit(mantissa * pow(2.0L, cast(real) exp), 123.456L, 19));
+    assert(approxEqual(mantissa * pow(2.0L, cast(real) exp), 123.456L));
```

Comment:
Yes, but does it pass on ARM? Or any other real==double target for that matter?

If equalsDigit and approxEqual are sufficiently the same, then there's no problem. :)